recipes/napari-workflows/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "napari-workflows" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-workflows-{{ version }}.tar.gz
+  sha256: 8f1ee8bbb31007a9da94300e9a51075a6eeeaf83f1dc681d8c47c2148f785d27
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
```

Comment:
This is not a proper plugin, right? Hence why the plugin engine dependencies are missing? But it's still installable in the UI?

Reply:
AFAIK it cannot be installed via napari plugin installer. This is no napari plugin.